### DISCUSSION TITLE
Start playback at a specific time

### DISCUSSION THREAD

**Discussion by yasaad** [vidstack/player#1109] [Q&A]
I am creating an audio application and when I switch between sources I want to start the audio where the user last left off. I tried adding remote.seek after changing the audio src but this doesn't seem to be consistent. Any suggestions on how to set the initial playback spot when I load in new audio?

**Comment by mihar-22** [comment]
We have a new storage solution coming in the next release which will make this really easy to do. It should be dropping in 2-3 days. You'll be able to save the info locally or in your own database. I'll share more in the release notes.

**Reply by KumarAnurag03** [reply]
Hello @mihar-22 ,

I am new to vidstack. I wanted to use the same feature, but for the video player. Is there a way that a particular video starts from the point where it is left? Is the feature functional now? Please guide me how to use it

**Reply by mihar-22** [reply]
Check out the docs @KumarAnurag03:

- [Web Components: Loading > Local Storage](https://www.vidstack.io/docs/wc/player/core-concepts/loading#storage)
- [React: Loading > Local Storage](https://www.vidstack.io/docs/player/core-concepts/loading#storage)

Simply set the `storage` prop or attribute and it will save user settings.

**Reply by KumarAnurag03** [reply]
Thanks a lot for your reply!!
It works fine now, i am able to retract to the previous quality/volume/time where i left. 
my local storage shows all the data , but not showing the time to send it to the backend. I want to know if there is a way to link the storage data(Time) with user id and video id?
